opencti-platform/opencti-front/lang/front/en.json

```
@@ -2982,6 +2982,7 @@
   "The Max Confidence Level is currently defined at the user level. It overrides Max Confidence Level from user's groups.": "The Max Confidence Level is currently defined at the user level. It overrides Max Confidence Level from user's groups.",
   "The Max Confidence Level is currently inherited from...": "The Max Confidence Level is currently inherited from the group {link}.",
   "The min value cannot be greater than max value": "The min value cannot be greater than max value",
+  "The NLQ model didn't find filters corresponding to your question": "The NLQ model didn't find filters corresponding to your question",
```

I'm not sure that NLQ is an "end user" word, at least it should be the whole word like :
```suggestion
  "The NLQ model didn't find filters corresponding to your question": "The Native Language Query model didn't find filters corresponding to your question",
```